assets/js/modules/reader-revenue-manager/datastore/publications.js

```diff
@@ -260,7 +307,6 @@ const baseResolvers = {
 			.getPublications();
 		if ( publications === undefined ) {
 			yield fetchGetPublicationsStore.actions.fetchGetPublications();
-			yield baseActions.syncPublicationOnboardingState();
```

Comment:
This has been removed based on the discussion [here](https://github.com/google/site-kit-wp/pull/9017#discussion_r1693356104).